src/it/java/com/google/checkstyle/test/chapter7javadoc/rule713atclauses/NonEmptyAtclauseDescriptionTest.java

```diff
@@ -8,54 +10,45 @@
 
 public class NonEmptyAtclauseDescriptionTest extends BaseCheckTestSupport{
 
-    private static ConfigurationBuilder builder;
-
-    @BeforeClass
-    public static void setConfigurationBuilder() {
-        builder = new ConfigurationBuilder(new File("src/it/"));
-    }
-
-    @Test
-    public void noFinalizerBasicTest() throws Exception {
-        
-        String msg = getCheckMessage(NoFinalizerCheck.class, "avoid.finalizer.method");
-
-        final String[] expected = {
-            "5: " + msg,
-        };
-
-        Configuration checkConfig = builder.getCheckConfig("NoFinalizer");
-        String filePath = builder.getFilePath("NoFinalizerInput");
-        
-        Integer[] warnList = builder.getLinesWithWarn(filePath);
-        verify(checkConfig, filePath, expected, warnList);
-    }
-    
     @Test
-    public void noFinalizerExtendedTest() throws Exception {
-        
-        String msg = getCheckMessage(NoFinalizerCheck.class, "avoid.finalizer.method");
+    public void testDefault() throws Exception {
+        ConfigurationBuilder builder = new ConfigurationBuilder(new File("src/it/"));
+        String msg = getCheckMessage(NonEmptyAtclauseDescriptionCheck.class, "non.empty.atclause");
 
         final String[] expected = {
-            "9: " + msg,
-            "21: " + msg,
-            "33: " + msg,
-            "45: " + msg,
-            "57: " + msg,
-            "69: " + msg,
+            //this is a case with description that is sequences of spaces
+            "27: " + msg,
+            //this is a case with description that is sequences of spaces
+            "28: " + msg,
+            //this is a case with description that is sequences of spaces
+            "29: " + msg,
+            //this is a case with description that is sequences of spaces
+            "38: " + msg,
+            //this is a case with description that is sequences of spaces
+            "39: " + msg,
+            //this is a case with description that is sequences of spaces
+            "40: " + msg,
+            "76: " + msg,
+            "77: " + msg,
+            "78: " + msg,
             "79: " + msg,
-            "119: " + msg,
-            "136: " + msg,
+            "80: " + msg,
+            "81: " + msg,
+            "90: " + msg,
+            "91: " + msg,
+            "92: " + msg,
+            "93: " + msg,
+            "94: " + msg,
         };
 
-        Configuration checkConfig = builder.getCheckConfig("NoFinalizer");
-        String filePath = builder.getFilePath("NoFinalizeExtendInput");
+        Configuration checkConfig = builder.getCheckConfig("NonEmptyAtclauseDescription");
+        String filePath = builder.getFilePath("InputNonEmptyAtclauseDescriptionCheck");
         
-        Integer[] warnList = builder.getLinesWithWarn(filePath);
+        Integer[] warnList = {27, 28, 29, 38, 39, 40, 76, 77, 78, 79, 80, 81, 90, 91, 92, 93, 94};
```

Comment:
> `{27, 28, 29, 38, 39, 40, 76, 77, 78, 79, 80, 81, 90, 91, 92, 93, 94}`

Can we do something about it? Looks like duplication - any change in input file would force us to update line numbers in two places.